spyder/plugins/remoteclient/widgets/connectiondialog.py

```diff
@@ -426,28 +424,127 @@ def _create_configfile_subpage(self):
         configfile = self.create_browsefile(
             text=_("Configuration file *"),
             option=f"{self.host_id}/configfile",
+            tip=_("File with the OpenSSH client configuration to use"),
             alignment=Qt.Vertical,
             status_icon=ima.icon("error"),
         )
 
+        host = self.create_lineedit(
+            text=_("Host *"),
+            option=f"{self.host_id}/{AuthenticationMethod.ConfigFile}/address",
+            default="",
+            tip=_(
+                "This is the host (<i><b>Host</b></i> configuration keyword) "
+                "that encapsulates the options to use for the connection"
+            ),
+            status_icon=ima.icon("error"),
+        )
+
+        port = self.create_spinbox(
+            prefix=_("Port"),
+            suffix="",
+            option=f"{self.host_id}/{AuthenticationMethod.ConfigFile}/port",
+            min_=0,
+            max_=65535,
+            tip=_("Introduce a port to use for this connection. "
+                  "Set <i><b>0</b></i> (<i><b>From file</b></i> value) to use "
+                  "the port specified in the configuration file"),
```

Comment:
```suggestion
            tip=_(
                "Introduce a port to use for this connection. Set <b>0</b> "
                "to use the port specified in the configuration file"
            ),
```

The problem with using some text in the spinbox is that it could be too long when localized to other languages. That's why I think we should use an empty string instead (see below).